app/src/main/java/com/example/foody/recipe_search/presentation_mvvm/ui/SearchScreen.kt

```diff
@@ -121,27 +146,42 @@ private fun SearchScreenContent(
     state: SearchScreenState,
     performSearch: () -> Unit,
     navigateWith: (recipeId: String) -> Unit,
-    onValueChanged: (searchTerm: String) -> Unit
+    onValueChanged: (searchTerm: String) -> Unit,
+    onFabClick: () -> Unit
 ) {
-
+    // for dimming the background ...
+    val modifier = if (state.searchBarState.expandedState) {
```

Comment:
can we `remember {` this?

Reply:
Yes, but do I need remember on this? ... ok, maybe I do.

Reply:
think this is now the only thing left to do...

Reply:
Applied in the next PR